cloud-formation/cloud-formation.yaml

```diff
@@ -41,13 +36,12 @@ Parameters:
     Type: String
     Default: t2.small
     AllowedValues:
-      - t2.micro
       - t2.small
     ConstraintDescription: must be a valid EC2 instance type.
   AMIID:
     Type: String
     Description: Amazon Machine Image ID
-    Default: ami-6857e51b
+    Default: ami-094aad897465c769e
```

Comment:
Does it make sense to apply a default here? I don't think so, because we'd never want to revert to an old AMI, and now that unused AMIs are cleaned up, this will id become obsolete in a few weeks.